### PR TITLE
Add assign-user functionality

### PR DESCRIPTION
**Pull request by paulbreuler**
Added initial functionality of assign-user to action repo.

**NOTE**: 

- PP-CLI-Wrapper version is bumped to "0.1.48" in this PR. Please reply if this needs to be changed. 
- NugetInstall now targets 1.14.x from daily build feed for Microsoft.PowerApps.CLI Windows and Linux variations.

### CODE REVIEW

**Comment by paulbreuler** [comment]
--team-id parameter was added to create-environment, this needs to be added in a separate PR to bring tests back to working order.

[Test fail due to create-environment missing param team-id](https://github.com/microsoft/powerplatform-actions/runs/5990583398?check_suite_focus=true#:~:text=src%5Cactions%5Ccreate,of%20type%20%27CreateEnvironmentParameters%27.)

**Review by petrochuk — APPROVED** [review]

**Comment by paulbreuler** [comment]
> --team-id parameter was added to create-environment, this needs to be added in a separate PR to bring tests back to working order.
> 
> [Test fail due to create-environment missing param team-id](https://github.com/microsoft/powerplatform-actions/runs/5990583398?check_suite_focus=true#:~:text=src%5Cactions%5Ccreate,of%20type%20%27CreateEnvironmentParameters%27.)
> 
> `src\actions\create-environment\index.ts(25,44): error TS2345: Argument of type '{ credentials: ClientCredentials | UsernamePassword; environmentName: HostParameterEntry; environmentType: HostParameterEntry; ... 4 more ...; domainName: HostParameterEntry; }' is not assignable to parameter of type 'CreateEnvironmentParameters'.`

fixed in PR #199

**Review by tehcrashxor — APPROVED** [review]

**Review by davidjenni — CHANGES_REQUESTED** [review]
couple questions/asks:
- please add your new task to the rolling-instance-actions flow and dispatch a run with your branch (setting the "slow actions" parameter to "no"). Would be great to have this part of this PR to assert the action succeeds.
- add a separate commit to your PR that updates the /dist folder (and gulpfile.js to pull in a daily pac CLI nupkg)
- until we lift the feature flag on your new verb, this repo for the time being cannot consume pac CLI from public nuget, but instead needs to pull from the daily build internal nuget feed
- should we flag the whole action and its 'object-id' and 'role' inputs as "preview" since there's already future work in your backlog to allow either guids to be specified as friendly name. This will turn the existing 2 inputs into optional. Other option is for this PR to hold, but I'm not fond of holding back PRs.

**Comment by paulbreuler** [comment]
> * should we flag the whole action and its 'object-id' and 'role' inputs as "preview" since there's already future work in your backlog to allow either guids to be specified as friendly name. This will turn the existing 2 inputs into optional. Other option is for this PR to hold, but I'm not fond of holding back PRs. **(DONE)**

Just look at those beautiful green check marks on those workflows 😊👀

File: gulpfile.js [hunk 1]
```diff
@@ -197,7 +197,7 @@ async function addDistToIndex() {
     console.log(`stderr: ${res.stderr}`);
 }
 
-const cliVersion = '1.14.2';
+const cliVersion = '1.14.3-daily-22041503';
```
Comment (davidjenni) [inline gulpfile.js:200]: update to latest Stable release 1.15.6

**Review by davidjenni — CHANGES_REQUESTED** [review]
reminder to also refresh dist w/ `npm run update-dist` again

**Comment by paulbreuler** [comment]
Holding for update from PAC CLI with latest bits

**Comment by paulbreuler** [comment]
closed due to avoid need to work through churn... See new PR #234

**Pull request closed without merging**